airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -466,6 +478,22 @@ public abstract AutoCloseableIterator<JsonNode> queryTableFullRefresh(final Data
                                                                         final String schemaName,
                                                                         final String tableName);
 
+  /**
+   * Read all data from a table.
+   *
+   * @param database source database
+   * @param columnNames interested column names
+   * @param schemaName table namespace
+   * @param tableName target table
+   * @param cursorField cursor field name
+   * @return iterator with read data
+   */
+  public abstract AutoCloseableIterator<JsonNode> queryTableFullRefresh(final Database database,
```

Comment:
And here, did you do this for the sake of demonstration?

Reply:
Again, I added the function so that the definition takes place in the top-level classes and the actual Implementation can take place in one of the special connectors